minibroker/setup-bucket.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+set -e
+set -x
+
+s3cmd mb s3://minibroker-helm-charts
+
+aws s3api put-bucket-versioning --bucket minibroker-helm-charts --versioning-configuration Status=Enabled
+
+cat <<EOF > index.yaml
+apiVersion: v1
+entries: ~
+generated: 2018-06-28T21:56:27.538677094Z
```

Comment:
Using an obviously-fake initial timestamp here would let people know that the empty file is because it hasn't ran before, not because it ran and gave nothing. (1970 is a nice year)

Reply:
Fixed in 0442755